src/__tests__/utils/test-utils.tsx

```diff
@@ -6,15 +6,15 @@ import { Provider } from 'react-redux';
 import { Router } from 'react-router-dom';
 
 import SnackbarCloseButton from '../../components/Shared/SnackbarCloseButton';
-import useProtocolTheme from '../../theme/protocolTheme';
+import getProtocolTheme from '../../theme/protocolTheme';
 import { store } from './setupTests';
 
 type ChildrenProps = { children: React.ReactElement };
 
 type ThemeConfig = Partial<Theme> | null;
 export function render(ui: React.ReactElement, themeConfig?: ThemeConfig) {
   function Wrapper({ children }: ChildrenProps) {
-    const { protocolTheme } = useProtocolTheme();
+    const { protocolTheme } = getProtocolTheme('light');
```

Comment:
@julienw Here I'm not sure how I can avoid hardcoding if the approach from the last two commits is alright. Let me know what you think. Thanks.

Reply:
I'm fine with that, especially that can be overriden by themeConfig. I'm not sure where themeConfig is used in tests currently (couldn't find it with a simple search), so if we need anything better in the future we can also revisit this parameter.